qcodes/instrument_drivers/devices.py

```diff
@@ -0,0 +1,81 @@
+from typing import Union
+
+from qcodes import Parameter, StandardParameter
+
+
+class VoltageDivider(Parameter):
+    """
+    Resitive voltage divider
+
+    To be used when you use a voltage divider to measure a parameter.
```

Comment:
Are you measuring the parameter. The example sounds more like you are setting an output?

Output of dac is X -> Voltage divider -> x/10 to actual Experiment